config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -70,9 +87,17 @@ presubmits:
         - --warnings=unknown-fields
         - --warnings=duplicate-job-refs
         - --warnings=unknown-fields-all
+        resources:
+          requests:
+            cpu: "1"
```

Comment:
we can try this but I think it's going to throttle excessively (we don't have GOMAXPROCS here)

Reply:
This is a very simple job that takes ~5 seconds and just runs existing binary in the image, I don't think CPU is much needed there. I could be mistaken though. https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/29617/pull-test-infra-prow-checkconfig/1663047464376078336